### ISSUE TITLE
How will the Rust Foundation interact with the All Hands Event? 

### ISSUE THREAD

**Issue by wesleywiser** [rust-lang/foundation-faq-2020#45]
First off, I'd just like to say thanks for doing this Q&A. It's been very insightful! 

I know the topic of Rust Conferences has been covered in #37 but I'd like to ask about the Rust All-Hands event specifically. The 2020 event ran into a number of obstacles especially around [financing the event and finding a bank account to collect and distribute funds from](https://blog.rust-lang.org/inside-rust/2020/03/18/all-hands-retrospective.html#budgeting-and-sponsorships). It seems like the Rust Foundation might be able to resolve one or both of those issues. 

Is supporting this event in the scope of the Foundation's activities?

**Comment by Mark-Simulacrum** [comment]
Yes! Events targeted at maintainers are definitely in scope, and we’d love to go beyond just a yearly All-Hands. One example from the past is Impl Days, and we’d enjoy running it again.
